src/mavlink_receive.c

```diff
@@ -125,10 +125,10 @@ static void mavlink_handle_msg_offboard_control(const mavlink_message_t *const m
     _offboard_control.y.type = PASSTHROUGH;
     _offboard_control.z.type = PASSTHROUGH;
     _offboard_control.F.type = THROTTLE;
-    _offboard_control.x.value = mavlink_offboard_control.x/2.0f;
-    _offboard_control.y.value = mavlink_offboard_control.y/2.0f;
-    _offboard_control.z.value = mavlink_offboard_control.z/2.0f;
-    _offboard_control.F.value = mavlink_offboard_control.F;
+    _offboard_control.x.value = mavlink_offboard_control.x*500.0f;
```

Comment:
So I think it'd be good at some point to move the mixer to floating point as well just so that everything can get passed through in normalized units to keep things consistent rather than carrying things around in PWM units (to me it seems like the PWM units probably shouldn't show up until as close to the actual hardware as possible). But let's make that a separate issue and not worry about it yet here.